meta-lmp-base/classes/uboot-fitimage.bbclass

```diff
@@ -56,11 +58,11 @@ uboot_fitimage_assemble() {
 	if ${ATF_SUPPORT}; then
 		optee_type="standalone"
 		config_firmware="atf"
-		config_loadables='"uboot", "optee"';
+		config_loadables='"uboot", "optee", "bootscr"';
 	else
 		optee_type="firmware"
 		config_firmware="optee"
-		config_loadables='"uboot"';
+		config_loadables='"uboot", "bootscr"';
 	fi
```

Comment:
Then you could add another if block based on BOOTSCR_SUPPORT and extend config_loadables as needed.